### PR TITLE
Pin perl version in perl-threaded

### PR DESCRIPTION
**Pull request by nsoranzo**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

xref. https://github.com/bioconda/bioconda-recipes/pull/3349/files#r95579261

### CODE REVIEW

File: recipes/perl-threaded/meta.yaml [hunk 1]
```diff
@@ -5,11 +5,11 @@ package:
   version: 5.22.0
 
 build:
-  number: 11
+  number: 12
 
 requirements:
   run:
-    - perl
+    - perl 5.22.0.1
```
Comment (johanneskoester) [inline recipes/perl-threaded/meta.yaml]: I guess we can leave out the `.1`.

Comment (nsoranzo) [inline recipes/perl-threaded/meta.yaml]: Fixed.

**Comment by bgruening** [comment]
Thanks Nicola!

**Comment by nsoranzo** [comment]
Thank you!

**Pull request merged**